infra/modular-examples/aws-google-workspace/main.tf

```diff
@@ -325,40 +323,38 @@ module "psoxy-bulk-to-worklytics" {
   }, try(each.value.settings_to_provide, {}))
 }
 
-module "psoxy_lookup_tables_builders" {
+# BEGIN lookup builders
+module "lookup_output" {
```

Comment:
instead of a lambda for each lookup table, just an output bucket (with associated IAM policies, settings, etc).